melodies_monet/driver.py

```diff
@@ -1257,6 +1301,8 @@ def plotting(self):
             from .plots import satplots as splots,savefig
         else: 
             from .plots import surfplots as splots, savefig
+            from .plots import aircraftplots as airplots
```

Comment:
@rschwant seems like this line is necessary for aircraft plots to work, though it wasn't in develop

Reply:
I see, it got deleted with one of the satellite updates. Thanks for noticing this.